### DISCUSSION TITLE
CPU Maxing Out....Again

### DISCUSSION THREAD

**Discussion by TheStonedSpider** [WarmUpTill/SceneSwitcher#1267] [General]
Hi, yeah its doing it again. I thought it was an issue on my end, but after I thought I fixed the issue its now doing it yet again. I feel its not shutting down objects in the scenes after it moves onto the next scene. i.e. playing videos, alerts etc.. and they just keep playing and its causing the cpu to max out. Or it seems like that. I set everything I could to shut down when not in focus but that doesnt seem to help either. Its doing it after an hour like before. I havent had a chance to use the scene switcher since i thought i had it fixed until today.

**Comment by WarmUpTill** [comment]
Just for reference, this is the previous discussion thread regarding this topic: https://github.com/WarmUpTill/SceneSwitcher/discussions/1261

As mentioned in the other discussion thread, the advanced scene switcher is not changing anything in regards to how OBS handles sources being rendered / not rendered. (Or at least I am not aware of how it would do that)
So, I have difficulties advising you how to proceed.

I can offer to try to have a look at an OBS log, if you have one where the issue was present.

Also, can you please share the macros you are using?
Preferably by [exporting them to a file](https://github.com/WarmUpTill/SceneSwitcher/wiki/Saving-and-loading-settings#how-to-create-and-import-a-backup-of-your-settings).
